reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -113,7 +115,7 @@ class FreemarkerTemplateProcessor(
         templateFiles.forEach { file ->
             log.info { "Generating output file using template file '${file.absolutePath}'." }
```

Comment:
Would it be good to log the path to the generated output file as well?

Reply:
Done in a new commit.